packages/cashier/src/Components/__tests__/cashier-container.spec.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import CashierContainer from '../cashier-container';
+
+import { routes } from '@deriv/shared';
+
+const mockHistoryPush = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useHistory: () => ({
+        push: mockHistoryPush,
+    }),
+}));
+
+describe('<CashierContainer/>', () => {
+    let clearIframe;
+
+    beforeAll(() => {
+        clearIframe = jest.fn();
+    });
+
+    it('should render the component with iframe when iframe_url is true', async () => {
+        const url = `https://doughflow-test.4x.my/cashier/login.asp?Sportsbook=test&PIN=CR90000010&Lang=en&Password=20192eef9f1c8258354030632a913916fe331c8f&Secret=11eb12e081e93780306d7c8a7483748a&Action=DEPOSIT&udef1=EN&udef2=deriv&theme=light`;
+        const { queryByTestId, container } = render(<CashierContainer iframe_url={url} clearIframe={clearIframe} />);
+        const loaderEle = container.querySelector('.barspinner');
```

Comment:
```suggestion
        const loader_element = container.querySelector('.barspinner');
```
Variables should be in lowercase separated by `_`. Please also name the variables as descriptive as possible.